pontoon/sync/core.py

```diff
@@ -32,17 +33,19 @@ def sync_project(db_project, now):
         changeset.execute()
 
 
-def serial_task(timeout, **celery_args):
+def serial_task(timeout, lock_key="", base=None, **celery_args):
     """
     Decorator ensures that there's only one running task with given task_name.
     Decorated tasks are bound tasks, meaning their first argument is always their Task instance
     :param timeout: time after which lock is released.
+    :param lock_key: allows to define different lock for respective parameters of task.
     """
+    task_class = base or PontoonTask
```

Comment:
Why add the base argument? You could leave it in `celery_args` and it would function identically.